pom.xml

```diff
@@ -1330,9 +1330,9 @@
             </plugin>
```

Comment:
please squash this commit with the first commit

Reply:
Squashed